extensions/security/oauth/src/org/exist/security/realm/oauth/OAuthRealm.java

```diff
@@ -44,7 +44,7 @@
 import org.exist.security.internal.aider.GroupAider;
 import org.exist.security.internal.aider.UserAider;
 import org.exist.storage.DBBroker;
-import org.exist.util.function.Function2E;
+import com.evolvedbinary.j8fu.function.Function2E;
```

Comment:
Actually it's kinda the other way. I had always copied and pasted simpler versions of some functional utilities that I had into eXist, it got to the point where I was maintaining them in two places and the upstream (j8fu) is more complete, so I just decided to add the dependency instead of copy/pasting and doing things twice.